.buildkite/scripts/dockerhub_publish.sh

```diff
@@ -4,15 +4,27 @@ set -euo pipefail
 
 source .buildkite/scripts/shared.sh
 version=$(buildkite-agent meta-data get "version")
-channel=$(buildkite-agent meta-data get release-channel)
 
 if is_fake_release; then
   # This overrides IMAGE_NAME in docker-base.sh
   export IMAGE_NAME="habitat/fakey-mc-fake-face-studio"
+else
+  export IMAGE_NAME="habitat/default-studio"
 fi
 
+if [[ -z $DOCKER_LOGIN_USER ]]; then
+  echo "DockerHub user required" && exit 1
+fi
+
+if [[ -z $DOCKER_LOGIN_PASSWORD ]]; then
+  echo "DockerHub password required" && exit 1
+fi
+
+docker login -u="$DOCKER_LOGIN_USER" -p="$DOCKER_LOGIN_PASSWORD"
```

Comment:
Minor, but it's generally good to use the long options in scripts for maximum readability, even though `u` and `p` are probably pretty idiomatic.

```sh
docker login \
       --username="$DOCKER_LOGIN_USER" \
       --password="$DOCKER_LOGIN_PASSWORD"
```